zipkin-storage/elasticsearch-http/src/test/java/zipkin/storage/elasticsearch/http/HttpElasticsearchTestGraph.java

```diff
@@ -16,13 +16,35 @@
 import com.google.common.collect.ImmutableList;
 import okhttp3.OkHttpClient;
 import org.junit.AssumptionViolatedException;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.wait.HttpWaitStrategy;
 import zipkin.Component.CheckResult;
+import zipkin.internal.Lazy;
 import zipkin.internal.LazyCloseable;
 import zipkin.storage.elasticsearch.ElasticsearchStorage;
 
 public enum HttpElasticsearchTestGraph {
   INSTANCE;
 
+  public Lazy<String> endpoint =
+    new Lazy<String>() {
+
+      @Override
+      protected String compute() {
+        try {
+          // TODO test different ES versions by using other images from Docker Hub
+          GenericContainer container = new GenericContainer("elasticsearch:2")
```

Comment:
"elasticsearch:5" works as well, wasn't sure which one to use. Also, can be set dynamically, of course. Maybe two separate test suites to test different ES versions (even locally)